mypy/build.py

```diff
@@ -1272,7 +1272,8 @@ def __init__(self,
                  ancestor_for: 'State' = None,
                  root_source: bool = False,
                  ) -> None:
-        assert id or path or source is not None, "Neither id, path nor source given"
+        assert id is not None or path is not None or source is not None, \
+            "Neither id, path nor source given"
```

Comment:
You're probably right. I've never been taught any rules about English, I've just built up a matching engine from reading a lot. So let's leave this.